cmd/collector/app/span_processor.go

```diff
@@ -130,6 +154,11 @@ func (sp *spanProcessor) saveSpan(span *model.Span) {
 	sp.metrics.SaveLatency.Record(time.Since(startTime))
 }
 
+func (sp *spanProcessor) countSpan(span *model.Span) {
+	atomic.AddUint64(&sp.bytesProcessed, uint64(span.Size()))
+	atomic.AddUint64(&sp.spansProcessed, 1)
```

Comment:
need to make sure these fields are properly aligned. Consider using uber-go/atomic instead, which we already depend on.